src/App.tsx

```diff
@@ -1,12 +1,52 @@
 import React from 'react';
 import './App.scss';
+import { GoodsList } from './components/GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+interface State {
+  goods: Good[]
+}
 
-export default App;
+export class App extends React.Component<{}, State> {
+  state = {
+    goods: [],
+  };
+
+  loadGoods = async (callback: Promise<Good[]>) => {
```

Comment:
Ти в цю функцію вже результат виклику колбека передаєш, а не сам колбек
```suggestion
  loadGoods = async (loadedGoods: Promise<Good[]>) => {
```